docs/getting-started-guides/ubuntu-calico.md

```diff
@@ -88,7 +98,12 @@ sudo cp -f kubernetes/server/bin/kubectl /usr/bin
 
 2.) Install the `kubelet` systemd unit file and start the `kubelet`.
 
+On your master, download the `calico-kubernetes` repository, which contains the necessary configuration for this guide.
+
 ```
+wget https://github.com/projectcalico/calico-kubernetes/archive/master.tar.gz
```

Comment:
I think it's better to download the files as they are required rather than grabbing the whole repo. They can be placed in the right directory as they are downloaded.